src/io_uring.rs

```diff
@@ -894,6 +902,9 @@ pub const IORING_OFF_CQ_RING: u64 = sys::IORING_OFF_CQ_RING as _;
 #[allow(missing_docs)]
 pub const IORING_OFF_SQES: u64 = sys::IORING_OFF_SQES as _;
 
+/// Indicate that the fd passed to [`io_uring_register`] is the index of a registered ring fd.
+pub const IORING_REGISTER_USE_REGISTERED_RING: u32 = sys::IORING_REGISTER_USE_REGISTERED_RING as _;
```

Comment:
As I understand it, this constant doesn't work with the current API because `io_uring_register` take as `IoringRegisterOp` which is an enum and doesn't support additional flags being or'd in.

Would it work to add a new `io_uring_register_with` function which is just like `io_uring_register` except that it has an extra `flags: IoringRegisterFlags` argument, where `IoringRegisterFlags` is a new `bitflags` type that currently just contains one flag, `USE_REGISTERED_RING`, and then the implementation could or that flags into the opcode value?